examples/hdr/rs-hdr.cpp

```diff
@@ -75,20 +80,32 @@ int main(int argc, char * argv[]) try
     // turning ON the HDR with the above configuration 
     depth_sensor.set_option(RS2_OPTION_HDR_ENABLED, 1);
 
-    // Create a simple OpenGL window for rendering:
-    window app(1280, 720, "RealSense Capture Example");
+    //initialize parameters to set view's window 
```

Comment:
PLs move these params out of  main and into to global list to keep the demo focused on HDR feature